### ISSUE TITLE
Petstore image used in GE GraphQL documentation does not work on M1/M2 macs

### ISSUE THREAD

**Issue by DuncanDoyle** [k8sgateway/k8sgateway#8754]
### Version

main (1.16.x beta)

### Describe the requested changes

The example used in the Gloo Edge GraphQL documentation uses a container image, `openapitools/openapi-petstore`, that does not (seem to) work on M1/M2 macs.

The `openapitools/openapi-petstore` image used in that Deployment  does come up, the pod is running, but there are no logs at all. When I try to test that API by port-forwarding into the container (port 8080) and access the API using cURL, the port-forward errors out:
```
Forwarding from 127.0.0.1:8080 -> 8080
Forwarding from [::1]:8080 -> 8080
Handling connection for 8080
E0928 11:33:02.862975   31367 portforward.go:409] an error occurred forwarding 8080 -> 8080: error forwarding port 8080 to pod 6017b17addcda89c2c0e961949ea812b45ba0f5864ee1e7a2f4f86e213a993ef, uid : exit status 1: 2023/09/28 09:33:02 socat[16126] E connect(5, AF=2 127.0.0.1:8080, 16): Connection refused
error: lost connection to pod
```
A couple of our engineers mentioned that indeed the `openapi-petstore` does not work on M1/M2.

At a minimum, we should mention this in a note in the doc. Better would be to find an alternative example that does work on M1/M2, as users with these kinds of setups (e.g. Minikube on Docker Desktop on M1/M2) are now unable to go through this "getting started".

### Link to any relevant existing docs

https://docs.solo.io/gloo-edge/latest/guides/graphql/getting_started/simple_discovery/

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1207635280052738/1207994529682598) by [Unito](https://www.unito.io)

**Comment by Nadine2016** [comment]
Quick fix for 1.16 https://github.com/solo-io/gloo/commit/5d97307e02741f708e88826a70bbc246f59e103e. Will look into this more as part of the 1.17 release.

**Comment by Nadine2016** [comment]
closing as it is related to GraphQL and this has been deprecated.
